src/main/resources/templates/index.html

```diff
@@ -28,7 +28,7 @@
 <div class="text-center">
   <div class="form-row">
     <div class="col">
-      <select class="custom-select" id="inputGroupSelect01"  style="width: 400px">
+      <select class="custom-select" id="inputGroupSelect01">

```

Comment:
Good change here... the author is mixing linked and inline styles, which is generally not good practice.  Use linked whenever possible; inline if it's an exception case.